explorer/scan/discovery_test.go

```diff
@@ -253,4 +253,19 @@ func TestDiscoverAssets(t *testing.T) {
 		require.NoError(t, err)
 		assert.Len(t, discoveredAssets.Assets, 1)
 	})
+
+	t.Run("container platform discovery", func(t *testing.T) {
```

Comment:
not sure if that's the best place to put this test but it's definitely important to have such tests... let's keep it here for now and we can break off real integration tests to another place later

Reply:
oh.. this needs to use a builtin os provider in order to work...

Reply:
moved it to the os provider